pkg/cmd/create.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+// NewCmdCreate provides a cobra command wrapping CreateOptions.
+func NewCmdCreate(streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewCreateOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:           "create",
+		Short:         "Create a resource",
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		RunE: func(c *cobra.Command, args []string) error {
+			return fmt.Errorf("must specify resource, only supported job")
```

Comment:
```suggestion
			return errors.New("must specify resource, only supported job")
```

In this case, I think better to `errors.New`.
(you need a fix imports)

Reply:
Thanks! I'll fix it.